search_service/api/swagger_doc/dashboard/search_dashboard_filter.yml

```diff
@@ -0,0 +1,41 @@
+Dashboard search
+This is used by the frontend API to search dashboard information.
+---
+tags:
+  - 'search_dashboard_filter'
+paths:
+  /search_dashboard:
+    post:
+      summary: This is used by the frontend API to search dashboard information.
+      requestBody:
+        description: The json data passed from the frontend API to execute a search.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                index:
+                  type: string
+                page_index:
+                  type: integer
+                  schema:
+                    type: integer
+                    default: 0
```

Comment:
Mmm... Not sure what `schema` with default value `0` is for.

Reply:
copy paste error, will clean up.